lib/ansible/plugins/lookup/merge_variables.py

```diff
@@ -0,0 +1,161 @@
+# (c) 2020 Thales Netherlands
+# (c) 2021 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    name: merge_variables
+    author:
+      - Roy Lenferink (!UNKNOWN) <lenferinkroy@gmail.com>
```

Comment:
I copied this `(!UNKNOWN)` part from another plugin: https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/lookup/password.py#L13

Is this correct or should I fill this in with something?

Reply:
I checked and it looks like the `(!UNKNOWN)` is the GitHub user name: https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/lookup/csvfile.py#L9

I have updated this to be the GitHub username. If it's incorrect let me know and i'll fix it.